tests/unit/test_exceptions.py

```diff
@@ -79,6 +79,12 @@ def testCodeInvariants(self):
 
     def testInstantiation(self):
         for class_ in self._getExceptionClasses():
+            # some exceptions are becoming too complicated to instantiate
+            # like the rest of the exceptions; just do them manually
+            if class_ in [
+                    exceptions.RequestValidationFailureException,
+                    exceptions.ResponseValidationFailureException]:
+                continue
```

Comment:
Each `continue` corresponds to at least ten dead puppies...

Anyway, I think
```python
args = defaultValue
if class_ in exceptionList:
    args = someThingElse
# Run the test with args...
```
is far better than not bothering running the test at all for the classes that are slightly different.